public/index.html

```diff
@@ -49,7 +67,7 @@ <h2>Part I</h2>
         <div class="row">
           <h2>Part II</h2>
           <p>
-            Using the API, implement a method to create a new Transfer between two users. This method should take arguments corresponding to the attributes of the Transfer resource. Before creating the Transfer, your method should confirm that the funds are available in the origin Account. This method should return the new Transfer as an Object. To test, create a Transfer using the following set of data:
+            Using the API, implement a method to <a href="api-docs.html#transfers">create</a> a new Transfer between two users. This method should take arguments corresponding to the attributes of the Transfer resource. Before creating the Transfer, your method should confirm that the funds are available in the origin Account. This method should return the new Transfer as an Object. To test, create a Transfer using the following set of request data:
```

Comment:
Maybe instead of "to test..." it just says something more along the lines of "Here's an example of input data"?

The "This method should return the new Transfer as an Object." is extra here now I think, given the extra output presented below.

Should we include something along the lines of "assume the API will handle updating related resources" after the note about how they are expected to confirm the funds are available? Then we could even say "assume the API will handle updating related resources, or returning an error if there was another failure."

Reply:
This is an interesting question.... is it a transfer between two users or between two accounts? We had one candidate who actually got confused about this language.

Reply:
Accounts, good catch @lskalkos-ellevest .